doc/sphinx/source/recipes/recipe_monitor.rst

```diff
@@ -0,0 +1,254 @@
+comments to define each option
+
+.. code-block:: yaml
+
+   maps:
+      global: # Map name, choose a meaningful one
+         projection: PlateCarree # Cartopy projection to use
+         projection_kwargs: # Dictionary with Cartopy's projection kwargs.
+            central_longitude: 285
+         smooth: true # If true, interpolate values to get smoother maps. If not, all points in a cells will get the exact same color
+         lon: [-120, -60, 0, 60, 120, 180] # Set longitude ticks
+         lat: [-90, -60, -30, 0, 30, 60, 90] # Set latitude ticks
+         colorbar_location: bottom
+         extent: null # If defined, restrict the projection to a region. Format [lon1, lon2, lat1, lat2]
+         suptitle_pos: 0.87 # Title position in the figure.
+
+Each entry in ``variable`` correspond to a variable definitions.
+Use the default entry to apply generic options to al variables.
+See below a sample with comments to define each option
+
+.. code-block:: yaml
+
+   variables:
+      # Define default. Variable definitions completely override the default
+      # not just the values defined. If you want to override only the defined
+      # values, use yaml anchors as shown
+      default: &default
+         colors: RdYlBu_r # Matplotlib colormap to use for the co,orbar
+         N: 20 # Number of map intervals to plot
+         bad: [0.9, 0.9, 0.9] # Color to use when no data
+      pr:
+         <<: *default
+         colors: gist_earth_r
+         # Define bounds of the colorbar, as a list of
+         bounds: 0-10.5,0.5 # Set colorbar bounds, as a list or in the format min-max,interval
+         extend: max # Set extend parameter of mpl colorbar. See https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.colorbar.html
+      sos:
+         # If default is defined, entries are treated as map specific option.
+         # Missing values in map definitionas are taken from variable's default
+         # definition
+         default:
+            <<: *default
+            bounds: 25-41,1
+            extend: both
+         arctic:
+            bounds: 25-40,1
+         antarctic:
+            bounds: 30-40,0.5
+      nao: &nao
+         <<: *default
+         extend: both
+         # Variable definitions can override map parameters. Use with caution.
+         bounds: [-0.03, -0.025, -0.02, -0.015, -0.01, -0.005, 0., 0.005, 0.01, 0.015, 0.02, 0.025, 0.03]
+         projection: PlateCarree
+         smooth: true
+         lon: [-90, -60, -30, 0, 30]
+         lat: [20, 40, 60, 80]
+         colorbar_location: bottom
+         suptitle_pos: 0.87
+      sam:
+         <<: *nao
+         lat: [-90, -80, -70, -60, -50]
+         projection: SouthPolarStereo
+         projection_kwargs:
+            central_longitude: 270
+         smooth: true
+         lon: [-120, -60, 0, 60, 120, 180]
+
+Variables
+=========
+
+* Any, but the dimensionality should match the expected by each plot
```

Comment:
expected what?

Reply:
Does `Any, but the dimensions of the variable should match the ones expected by each plot` read better to you?